full_snapshot.proto

```diff
@@ -21,6 +21,11 @@ message FullSnapshot {
 
   CollectorStatistic collector_statistic = 20;
   repeated string collector_errors = 21; // log error messages that happened during the collector run
+  google.protobuf.Timestamp collector_started_at = 22;
+  string collector_os = 23; // coarse-grained, from runtime.GOOS
+  string collector_os_distro = 24; // e.g., ubuntu, amazon linux, redhat
+  string collector_os_version = 25; // distro-specific version string
```

Comment:
@lfittl Does this breakdown make sense? I know collector_os_distro is what's really useful, but it might also be good to separately also have more and less detail.

Reply:
I think that makes sense. Although I do wonder if we should match some of the terminology we have for system snapshots:

* https://github.com/pganalyze/collector-snapshot/blob/master/shared.proto#L133
* https://github.com/pganalyze/collector/blob/master/input/system/selfhosted/system.go#L51

Reply:
Yes, good call.